docs/networks/network-guides/switch-consensus-clients.md

```diff
@@ -91,13 +91,7 @@ After the initialization is successful, we must install the new blockchain clien
 
 :::tip
 
-It is recommended to use Erigon and Lighthouse clients to address both client majorities at once.
-
-:::
-
-:::info
-
-After the installation has been successful, you can continue importing your validator keys into the new validator wallet that will be used within Lighthouse. You will be asked to set a new wallet password to safely store the imported deposit keys. Further, you must input your password for the deposit keys to be encrypted and added to the wallet.
+To balance the network, it is recommended to use Erigon as execution and Lighthouse or Teku as consensus clients.
 
```

Comment:
Info box had doubled/cloned content. deleted text is written below next heading again